clients/index.js

```diff
@@ -417,3 +418,17 @@ ApplicationClients.prototype.updateKValues = function updateKValues() {
         self.serviceProxy.updateServiceChannels();
     }
 };
+
+ApplicationClients.prototype.updateKillSwitches = function updateKillSwitches() {
+    var self = this;
+    self.serviceProxy.unblockAll();
```

Comment:
Won't this remove any manual ones we run? I think it would still be useful to have an immediate fan out kill switch rather than relying on remote-config to load the changes.